### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by jajinmori1** [mlflow/mlflow#5558]
#%%
import json
import os
from sklearn.metrics import classification_report
import time
import mlflow
from mlflow.models.signature import infer_signature
from urllib.parse import urlparse
import pandas as pd
import glob
import pymysql
from lightgbm import LGBMClassifier
from sklearn.model_selection import train_test_split

mlflow.set_tracking_uri("###")
mlflow.get_tracking_uri()

PIP_REQUIREMENTS = [f"mlflow=={mlflow.__version__}"]
PIP_REQUIREMENTS

df = pd.read_csv("dataset\master_data_CHS.csv", encoding = 'euc-kr', index_col=0)
drop_columns = ['column1', 'APGAR5']
label = 'column1'
features = df.drop(drop_columns, axis=1).columns
X_train, X_test, y_train, y_test = train_test_split(df[features], df[label])

MODEL_NAME = 'LGBM_001'
EXPERIMENT_NAME = 'test_001'
OUTPUT_DIR = f"{MODEL_NAME}_{EXPERIMENT_NAME}_output"
MODEL_DIR = f"model"

EXPERIMENT_ID = mlflow.create_experiment(f"{MODEL_NAME}_{EXPERIMENT_NAME}")

with mlflow.start_run(experiment_id=EXPERIMENT_ID) as run:
  #! -------------------------------------------
  print(f"Model name: {MODEL_NAME}")
  RUN_ID = run.info.run_id
  print(f"Run id: {RUN_ID}")
  EXPERIMENT_ID = run.info.experiment_id
  print(f"Experiment id: {EXPERIMENT_ID}")

  #* -------------------------------------------
  print("Starting training...")
  model = LGBMClassifier(random_state=0) # parmameter 
  model.fit(X_train, y_train)

  tracking_url_type_store = urlparse(mlflow.get_tracking_uri()).scheme
  if tracking_url_type_store != "file":
    mlflow.lightgbm.log_model(model, f"{MODEL_NAME}_{EXPERIMENT_ID}_{RUN_ID}", registered_model_name=MODEL_NAME, pip_requirements=PIP_REQUIREMENTS)
  else:
    mlflow.lightgbm.log_model(model, f"{MODEL_NAME}_{EXPERIMENT_ID}_{RUN_ID}", pip_requirements=PIP_REQUIREMENTS)
  print("- Finished!")
# %%

I can't  see model artifacts in UI...
loading continues....
when I first complete this code I can see model artifacts in UI ,but suddenly I can't see any artifacts...

**Comment by harupy** [comment]
@jajinmori1 Do you log artifacts on your local machine or a remote artifact store like S3?

**Comment by jajinmori1** [comment]
I didnt. first I wanna see my model on my UI.

**Comment by harupy** [comment]
@jajinmori1 I think you did because your code contains `mlflow.lightgbm.log_model`. Can you check where your model artifacts are stored on UI?

<img width="1242" alt="image" src="https://user-images.githubusercontent.com/17039389/160967713-e7516d79-71e1-45d4-9c4f-8226440005f7.png">

**Comment by jajinmori1** [comment]
@harupy 
![image](https://user-images.githubusercontent.com/80096854/160975024-927fa4e5-99dd-4859-a396-05f9fe5a817c.png)
UI can't load arifacts.

**Comment by jajinmori1** [comment]
@harupy When I first Execute this cord, I can see model and artifacts just like u uploaded, but suddenly I can't see anything...

**Comment by harupy** [comment]
Could you open the dev console of your browser and see if there any error message s?

**Comment by jajinmori1** [comment]
@harupy there is no error message....T.T

**Comment by jajinmori1** [comment]
![image](https://user-images.githubusercontent.com/80096854/160977227-bd8c683c-b079-4b93-a36d-90ef93426b77.png)
@harupy do u wanna see this one?

**Comment by harupy** [comment]
Can you see params and metrics?

**Comment by jajinmori1** [comment]
@harupy I didn't load params and metrics, I just loaded model

**Comment by jajinmori1** [comment]
@harupy when I load params and metrics, I can see on UI, only model and pictures some kind of artifacts, I can't see them on UI

**Comment by harupy** [comment]
Is your tracking server logging something?

**Comment by jajinmori1** [comment]
@harupy I'm not sure, but it's public db(dompany team db)

**Comment by harupy** [comment]
I guess something is wrong with your tracking server. Could you ask the person who maintains the tracking server?

**Comment by jajinmori1** [comment]
@harupy Ok, I will try. thx sir

**Comment by harupy** [comment]
@jajinmori1 Are you still experiencing the issue?

**Comment by harupy** [comment]
@jajinmori1 I'll close this issue. Feel free to reopen if you need further investigation.
